Documentation/user-guides/cluster-monitoring.md

```diff
@@ -37,7 +37,7 @@ The manifests used here use the [Prometheus Operator](https://github.com/coreos/
 >
 > In future versions of minikube and kubeadm this will be the default, but for the time being, we will have to configure it ourselves.
 
-[embedmd]:# (../../contrib/kube-prometheus/manifests/prometheus-operator/prometheus-operator-deployment.yaml)
+[embedmd]:# (../../contrib/kube-prometheus/manifests/0prometheus-operator-deployment.yaml)
```

Comment:
What is the leading `0` needed for in `0prometheus-operator-deployment`? For load precedence?

Reply:
yes, that way we can just run `kubectl create -f manifests/`, which is a very smooth getting started experience I think